app/models/katello/content_view_history.rb

```diff
@@ -15,12 +15,15 @@ class ContentViewHistory < Katello::Model
     validates_lengths_from_database
     validates :status, :inclusion => {:in          => STATUSES,
                                       :allow_blank => false}
+    validates :action, presence: true, on: :create
 
     scope :active, -> { where(:status => IN_PROGRESS) }
     alias_method :version, :content_view_version
 
     scoped_search :on => :name, :in => :environment, :rename => :environment, :complete_value => true
 
+    enum action: [:publish, :promotion, :removal, :export]
```

Comment:
reading up on enums (as I haven't used them a whole lot), it seems safer to actually define their values (https://bendyworks.com/rails-enum-sharp-knife/)

any thoughts on that?